### ISSUE TITLE
100% cpu usage on one core

### ISSUE THREAD

**Issue by oomek** [microsoft/vscode#185462]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.79.2
- OS Version: Windows 11 ( build: 22631.1900 )

Steps to Reproduce:

1.  Open VSCode
2.  Open Taks manager

![image](https://github.com/microsoft/vscode/assets/2974860/4ead4ff3-04ce-4813-bdba-14cea0b49921)

Tried `code --disable-extensions`,
Tried uninstalling extensions.
Tried insiders build 1.80.0 fresh with 0 extensions installed.

One core is constantly maxed out. When I close VSCode the cpu usage goes back to normal.

`code --status` does not show anything abnormal

```
C:\APPS\TOOLS\VSCode>"code - insiders" -s

Version:          Code - Insiders 1.80.0-insider (61ba389b0a7108af0411c73ab30c7e4350051645, 2023-06-16T09:46:56.229Z)
OS Version:       Windows_NT x64 10.0.22631
CPUs:             Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (12 x 3696)
Memory (System):  15.95GB (8.55GB free)
VM:               0%
Screen Reader:    no
Process Argv:     --crash-reporter-id 3aa18054-73cf-41ba-b57a-fd8acfc07cdb
GPU Status:       2d_canvas:                              enabled
                  canvas_oop_rasterization:               disabled_off
                  direct_rendering_display_compositor:    disabled_off_ok
                  gpu_compositing:                        enabled
                  multiple_raster_threads:                enabled_on
                  opengl:                                 enabled_on
                  rasterization:                          enabled
                  raw_draw:                               disabled_off_ok
                  video_decode:                           enabled
                  video_encode:                           enabled
                  vulkan:                                 disabled_off
                  webgl:                                  enabled
                  webgl2:                                 enabled
                  webgpu:                                 enabled

CPU %   Mem MB     PID  Process
    0       93   24392  code-insiders main
    0       45    8008     utility-network-service
    0       82   10124  shared-process
    0       92   11656     gpu-process
    0      102   17032  extensionHost [1]
    0       29   20960     crashpad-handler
    0       75   21568  ptyHost
    0       74   22612  fileWatcher [1]
    0      163   24220  window [1] (Welcome - Visual Studio Code - Insiders)
```

Cpu usage after closing VSCode

![image](https://github.com/microsoft/vscode/assets/2974860/c683e150-0f81-461b-b7a1-f52030b911aa)

**Comment by oomek** [comment]
After 2 days this behaviour stopped and I have no idea why. I've done several reboots in the meantime but only this morning I can not longer reproduce the 100% cpu usage on of the cores.

**Comment by deepak1556** [comment]
@oomek can you clarify your last comment, is the issue still happening with your setup ?

**Comment by oomek** [comment]
The issue isn't happening anymore.

**Comment by deepak1556** [comment]
Thanks for confirming

**Comment by willeumanuel** [comment]
I'm facing exactly this problem since yesterday sunday, suddenly the cpu usage went up to 100%. I'm using version 1.79.2, when I don't type anything from VSCode the CPU is idle again. I'm here wondering what might be happening, I saw some recommendations on StackOverflow in this link: [stackOverflow/vs-code-uses-100-cpu-even-if-it-is-closed](https://stackoverflow.com/questions/71516186/vs-code-uses-100-cpu-even-if-it-is-closed)
And still the problem persists! :(
![Captura de Ecrã (7)](https://github.com/microsoft/vscode/assets/53792918/198f1369-97bd-4e0a-ab6c-8b53c0330014)

**Comment by virgiliofornazin** [comment]
I discovered how to reproduce this issue on Ubuntu 22.04 and latest VSCode running on Hyper-V.

I opened a project and did nothing, CPU usage is as expected.

![image](https://github.com/microsoft/vscode/assets/4605542/a015e7b6-7632-4ebb-898f-e07df883f6e8)

However, when I am building this project, CPU usage goes high... and I cannot make a correlation on that.
Is only drawing tooking too much CPU cycles?

![image](https://github.com/microsoft/vscode/assets/4605542/687770ed-58c0-450d-82d0-e3e865b6de6e)
